app/src/main/java/com/appunite/loudius/ui/pullrequests/PullRequestsViewModel.kt

```diff
@@ -33,12 +33,14 @@ sealed class PulLRequestsAction {
     object RetryClick : PulLRequestsAction()
 }
 
-data class PullRequestState(
-    val pullRequests: List<PullRequest> = emptyList(),
-    val navigateToReviewers: NavigationPayload? = null,
-    val isLoading: Boolean = false,
-    val isError: Boolean = false,
-)
+sealed class PullRequestState {
+    object Loading : PullRequestState()
+    object Error : PullRequestState()
+    data class Loaded(
+        val pullRequests: List<PullRequest> = emptyList(),
+        val navigateToReviewers: NavigationPayload? = null,
+    ) : PullRequestState()
```

Comment:
Suggestion: What do you think about `Success` instead of `Loaded` or something like that?

Reply:
suggestion: What do you think about:

```kotlin
sealed class Data {
  object Loading : Data()
  object Error : PullRequestState()
  data class Loaded(val pullRequests: List<PullRequest>)
}

data class PullRequestState(
  val data: Data,
  val navigateToReviewers: NavigationPayload? = null,
)
```

So `navigateToReviewers` is not related to the sealed class.